accounting/create_db.py

```diff
@@ -47,7 +48,27 @@ def create_db(filepath):
 
 
 def main():
-    create_db("FluxAccounting.db")
+
+    parser = argparse.ArgumentParser(
+        description="""
+        Description: Create flux-accounting database to store
+        user account information.
+        """
+    )
+
+    parser.add_argument(
+        "-p", "--path", dest="path", help="specify location of database file"
+    )
+
+    args = parser.parse_args()
+
+    if args.path:
+        try:
+            create_db(args.path)
+        except sqlite3.OperationalError:
+            print("Unable to create database file")
```

Comment:
If the database fails to be created, I think this script should exit with a non-zero exit code.